CMakeLists.txt

```diff
@@ -158,10 +158,12 @@ if (error EQUAL 0)
   ADD_SUBDIRECTORY(src/lib/orionTypes)
   ADD_SUBDIRECTORY(src/lib/ngsi)
   ADD_SUBDIRECTORY(src/lib/serviceRoutines)
+  ADD_SUBDIRECTORY(src/lib/serviceRoutinesV2)
```

Comment:
We were.
I believe it is better adding the version to the directory name instead.
This way all 'lib directories' are on the same level. More order ...